src/content/self-hosted/administration/configuration.mdx

```diff
@@ -1065,7 +1065,9 @@ prepullImages:
   enabled: true
 ```
 
-### privateRegistry
+### privateRegistry (deprecated)
```

Comment:
Let's remove this section, it's already documented for older versions

Reply:
This section is already removed in my PR about private registries: https://github.com/okteto/docs/pull/509/files#diff-aeb102e057b5bde09ec890e099345b697f54b1904aa565a02142aaa0eb9cfbb8

I wouldn't modify this to avoid conflicts between both PRs and mine is covering this